src/main/java/uk/gov/hmcts/reform/et/syaapi/constants/EtSyaConstants.java

```diff
@@ -46,6 +46,8 @@ public final class EtSyaConstants {
     public static final String SEND_EMAIL_SERVICE_OWNER_NAME_KEY = "serviceOwnerName";
     public static final String SEND_EMAIL_SERVICE_OWNER_NAME_VALUE = "Service Owner";
     public static final String SCOTLAND_CASE_TYPE = "ET_Scotland";
+    public static final DateTimeFormatter OLD_DATE_TIME_PATTERN =
+        DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS");
```

Comment:
Does this exist in et-common?

Reply:
It exists in ecm.common, so I am updating now. Thanks